src/Sarif.FunctionalTests/ConverterTestData/Semmle/results.csv.sarif

```diff
@@ -0,0 +1,1353 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-1.0.0",
+  "version": "1.0.0",
+  "runs": [
+    {
+      "tool": {
+        "name": "Semmle"
+      },
+      "results": [
+        {
+          "level": "warning",
+          "message": "Avoid using thread priorities. The effect is unpredictable and not portable.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/FileCleaningTracker.java",
+                "region": {
+                  "startLine": 211,
+                  "startColumn": 13,
+                  "endLine": 211,
+                  "endColumn": 44
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "warning",
+          "message": "Incorrect lazy initialization of static field [[\"SKIP_BYTE_BUFFER\"|\"relative://src/main/java/org/apache/commons/io/IOUtils.java:170:6561:16\"]]: Missing synchronization.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/IOUtils.java",
+                "region": {
+                  "startLine": 2718,
+                  "startColumn": 13,
+                  "endLine": 2718,
+                  "endColumn": 57
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "warning",
+          "message": "Incorrect lazy initialization of static field [[\"SKIP_CHAR_BUFFER\"|\"relative://src/main/java/org/apache/commons/io/IOUtils.java:169:6517:16\"]]: Missing synchronization.",
```

Comment:
I'll add to my mail to them, the question of whether there are restrictions on the links they embed.

---
In reply to: [90075185](https://github.com/Microsoft/sarif-sdk/pull/545#discussion_r90075185) [](ancestors = 90075185)